project/Dependencies.scala

```diff
@@ -34,6 +34,7 @@ object Dependencies {
     "org.apache.pekko" %% "pekko-persistence-query" % PekkoVersion,
     "com.typesafe.slick" %% "slick" % SlickVersion,
     "com.typesafe.slick" %% "slick-hikaricp" % SlickVersion,
+    "com.zaxxer" % "HikariCP" % "6.0.0",
```

Comment:
maybe add a comment that we can get this transitively again once we unpin slick?